changelog.md

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.19.2 (2021-05-28)
+-------------------
+
+* Fix: bug in JSON schema for group members. Inconsistent
+property names.
```

Comment:
```suggestion
* Fix: bug in JSON schema for group members. Inconsistent property
  names
```

Reply:
Just editing the space here. Newlines in a list need to start with 2 spaces in markdown